department-of-veterans-affairs/caseflow
# janet/APPEALS-13521

Resolves [APPEALS-13521](https://vajira.max.gov/browse/APPEALS-13521)

### Description
As a Caseflow Developer, I need to create data seeding script so that we have data to test the new Travel Board job in local, DEMO and UAT environments.

### Acceptance Criteria
- [ ] The Vacols data seed scripts are updated with vacols appeals needs travel board hearings scheduled
- [ ] A sql script exists that can be ran against UAT vacols to stage vacols appeals that need travel board hearings scheduled

### Testing Plan
1. Run the script
2. Check BRIEFF table for 30 new entries
3. Check FOLDER table for 30 new entries
4. Check ISSUES table for 30 new entries
